packages/cactus-odap-hermes/src/main/json/openapi.json

```diff
@@ -0,0 +1,1216 @@
+{
+    "openapi": "3.0.3",
+    "info": {
+        "title": "Hyperledger Cactus Plugin - Connector Besu",
+        "description": "Can perform basic tasks on a Besu ledger",
+        "version": "0.0.1",
+        "license": {
+            "name": "Apache 2.0",
+            "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+        }
+    },
+    "servers": [
+        {
```

Comment:
@jscode017 Please remove the `servers` array, it's not needed as we recently found out it actually causes bugs elsewhere in the code.

Reply:
ok, solve this with new commits